### ISSUE TITLE
Support `certbot help <command>` syntax

### ISSUE THREAD

**Issue by jsha** [certbot/certbot#3371]
This is a fairly common idiom in command line tools (at least in Go), and it's nice to be generous about giving help.

Right now if you run this you get the regular help, with an easy-to-miss error squished at the bottom:

```
certbot: error: unrecognized arguments: help
```

**Comment by pde** [comment]
This would be an extra call to `HelpfulArgumentParser.prescan_for_flag` plus some more processing of the results in case the user does something nuts like `certbot help --help help -h standalone` :)

**Comment by ynasser** [comment]
I've started working on this issue. Will post updates/questions as they arise :)

**Comment by bmw** [comment]
Fixed in #4068.

By the way, only the `certbot help ...` syntax works. If `help` is in any other position, it's not treated specially. The reason for this is we can't (sanely) tell whether `help` means `--help` or the user is passing `help` to an argument. For example:
```
certbot --third-party-plugin-flag help
```
How should Certbot handle this? Nobody knows...
